[docs] Improve Issue/PR templates (#11499)
yt-dlp/yt-dlp: .github/PULL_REQUEST_TEMPLATE.md

<!--
    **IMPORTANT**: PRs without the template will be CLOSED
    

>     Due to the high number of pull requests, it may take some time before your code will be reviewed.
>     It is recommended for new contributors to stick to smaller pull requests, to familiarize themselves with the codebase.
>     Large PRs take longer to be reviewed and merged.

Reviewer: maybe something like this?
```suggestion
    Due to the high volume of pull requests, it may be a while before your PR is reviewed.
    Please try to keep your pull request focused on a single bugfix or new feature.
    Pull requests with a vast scope and/or very large diff will take much longer to review.
    It is recommended for new contributors to stick to smaller pull requests, so you can receive much more immediate feedback as you familiarize yourself with the codebase.
```

>     Please avoid force-pushing once a review has started, as it makes reviewing more difficult.

Reviewer: ```suggestion
    PLEASE AVOID FORCE-PUSHING once a review has started, as it makes reviewing more difficult.
```

<!--

>     **IMPORTANT**: PRs without the template will be CLOSED

Reviewer 2: This was kept outside comment so it's visible in preview

Author: As we are expending that section, I think it makes sense to have this in a comment instead of cluttering up the actual PR. It is the very first thing in the PR Template. You would have to be wilfully ignorant to ignore it, at which point the text being present in the preview won't help things.

    
    Due to the high volume of pull requests, it may be a while before your PR is reviewed.
    Please try to keep your pull request focused on a single bugfix or new feature.
    Pull requests with a vast scope and/or very large diff will take much longer to review.
    It is recommended for new contributors to stick to smaller pull requests, so you can receive much more immediate feedback as you familiarize yourself with the codebase.

>     PLEASE AVOID FORCE-PUSHING once a review has started, as it makes reviewing more difficult.

Reviewer 2: I've lost draft review as well
```suggestion
    PLEASE AVOID FORCE-PUSHING once a PR is opened, as it makes reviewing more difficult.
```

Reviewer 3: Maybe this?

```suggestion
    PLEASE AVOID FORCE-PUSHING once you opened a PR, as it makes reviewing more difficult.
```

Contributors see this notification when they are opening a new PR.
